deploy.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 gcloud config set project llvm-build-dashboard &&
-mvn package com.google.cloud.tools:appengine-maven-plugin:deploy
+mvn clean package &&
```

Comment:
a suggestion for a slight improvement: you can start the bash script with [`set -e`](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script) and remove all the trailing `&&`-s. This set option has the effect of exiting as soon as the first command executed returns a non-zero exit status.

Reply:
Awesome!